lib/src/equatable_utils.dart

```diff
@@ -46,6 +48,11 @@ int _combine(int hash, dynamic object) {
     });
     return hash;
   }
+  if (object is Set) {
```

Comment:
should we also check to make sure `object` is not already a `SplayTreeSet` to avoid unnecessarily created a new instance?
```suggestion
  if (object is Set && object is! SplayTreeSet) {
```

Reply:
Good point!